widgets/nuxeo-textarea.js

```diff
@@ -70,6 +70,7 @@ import '@polymer/paper-input/paper-textarea.js';
       readonly\$="[[readonly]]"
       error-message="[[errorMessage]]"
       validator\$="[[validator]]"
+      placeholder\$="[[placeholder]]"
```

Comment:
We shouldn't need the \$ here. I believe this is an artefact generate by polymer modulizer.

Reply:
Cleaned for all properties.